packages/plugin/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@package-inspector/plugin",
```

Comment:
as discussed, renaming would be beneficial to indicate what the package is actually used for, something like `@package-inspector/plugin-preset` would provide clearer intent

Reply:
updated